data/mods/ssb/moves.ts

```diff
@@ -72,6 +72,68 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Normal",
 	},
 
+	// Jett
```

Comment:
```suggestion
	// Crystalites
```

Reply:
He's gonna change his name to Jett

Reply:
> Name: Crystalites (I prefer "Jett" since I plan to change my ps name in the next couple months)

Reply:
[3:46 PM] Kris: @Spandan im aware he is changing his name to jett
[3:46 PM] Kris: however until that is his PS name, we arent using it
[3:46 PM] Kris: luckily he is changing it later today